command/utils.go

```diff
@@ -0,0 +1,12 @@
+package command
+
+import (
+	"fmt"
+
+	"gopkg.in/alecthomas/kingpin.v2"
```

Comment:
Import statements should be grouped by their domain and separated by a blank line. Consider putting the 'fmt' import in a separate group.